### DISCUSSION TITLE
Vistiting some pages returns a 404 despite the HTML file existing

### DISCUSSION THREAD

**Discussion by tomaka** [community/community#55035] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi :wave: 

I am encountering a problem, and I am a bit puzzled by it.

I have the GitHub pages feature activated for my repo (https://github.com/smol-dot/smoldot), with an automatic deployment from the `gh-pages` branch. Nothing out of the ordinary.

Most of the website works. For example, one of the pages is https://smol-dot.github.io/smoldot/tests-coverage/index.html, and you can see it in the repo here: https://github.com/smol-dot/smoldot/tree/gh-pages/tests-coverage.

However, all of the links on the page above returns a 404, and I don't understand why.
For example, the URL https://smol-dot.github.io/smoldot/tests-coverage/coverage/__w/smoldot/smoldot/repo/full-node/src/cli.rs.html returns a 404, even though the file exists in the repo: https://github.com/smol-dot/smoldot/blob/gh-pages/tests-coverage/coverage/__w/smoldot/smoldot/repo/full-node/src/cli.rs.html

Basically, any URL under https://github.com/smol-dot/smoldot/tree/gh-pages/tests-coverage/coverage seems to return a 404.

Note that these pages are automatically generated by a tool (`llvm-cov` to name it), so I expect the link to be correct, and even if they weren't, I have manually double-checked the URLs, and I can't see what's wrong.

Is there something that I'm missing? Or is it a weird GitHub bug?

Thanks in advance

**Comment by mrepol742** [comment, marked as answer]
Try adding a `.nojekyll` file to your gh-pages root directory it seems like not just files that starts with dot but it also ignores files that begins with underscore.

**Reply by tomaka** [reply]
Ah thanks, that fixed it!
